File: lib-util/src/main/java/code/util/NumberUtil.java
package code.util;

public class NumberUtil {

    public static int parseInt(String intText) {
        int intValue = 0;
        try {
            intValue = Integer.parseInt(intText);
        } catch (Exception e) {
        }
        return intValue;
    }

    public static long parseLong(String longText) {
        long longValue = 0;
        try {
            longValue = Long.parseLong(longText);
        } catch (Exception e) {
        }
        return longValue;
    }

    public static float parseFloat(String floatText) {
        float floatValue = 0;
        try {
            floatValue = Float.parseFloat(floatText);
        } catch (Exception e) {
        }
        return floatValue;
    }
}

File: lib-util/src/main/java/code/util/NumberUtil.kt
package code.util

object NumberUtil {

    fun parseInt(intText: String?): Int {
        var intValue = 0
        try {
            intValue = intText!!.toInt()
        } catch (_: Exception) {
        }
        return intValue
    }


    fun parseLong(longText: String?): Long {
        var longValue: Long = 0
        try {
            longValue = longText!!.toLong()
        } catch (_: Exception) {
        }
        return longValue
    }

    fun parseFloat(floatText: String?): Float {
        var floatValue = 0f
        try {
            floatValue = floatText!!.toFloat()
        } catch (_: Exception) {
        }
        return floatValue
    }
}